views/js/qtiCommonRenderer/renderers/interactions/AssociateInteraction.js

```diff
@@ -381,24 +381,46 @@ define([
             // Point & click handlers
 
             interact($container.selector).on('tap', function() {
+                //if tts component is loaded and click-to-speak function is activated - we should prevent this listener to go further
+                if ($(e.currentTarget).parents('.qti-item').hasClass('prevent-click-handler')) {
```

Comment:
Does the element have several parents with the class `.qti-item` ? Otherwise it should be better to use `.closest()` method.

Reply:
It is a question not to discuss, but make it clearer for me:
Do we have only one item at a time? I mean that do item with two/three/more interactions inside have only one `.qti-item` container? I'll test this situation a little bit later, if yes - this code will be left as is at the moment, if no - will be changed to `closest`

Reply:
Yes, our implementation of the TestRunner only displays one item at a time.
An item may contains more that one interaction, but the item container is the same for all. Each interaction have its own sub container `.qti-interaction`.